$NetBSD: patch-al,v 1.2 2014/04/26 13:51:40 wiz Exp $

Echogs is almost strings command and accepts some glue to output
from its command line. The output, in this case ldt.tr, will be really
shell script for ld command. The adding words with this patch,
$(LDFLAGS_SO), will be included in the argument of ld command.

$(LDFLAGS_SO) is defined in patch-af, for Mac OS X, Darwin.
 -- added by mef@ --

--- base/unixlink.mak.orig	2010-02-18 12:08:31.000000000 +0000
+++ base/unixlink.mak
@@ -50,7 +50,7 @@ $(GS_A): $(obj_tr) $(ECHOGS_XE) $(INT_AR
 # which has limited environment space.
 ldt_tr=$(PSOBJ)ldt.tr
 $(GS_XE): $(ld_tr) $(ECHOGS_XE) $(XE_ALL) $(PSOBJ)gsromfs$(COMPILE_INITS).$(OBJ)
-	$(ECHOGS_XE) -w $(ldt_tr) -n - $(CCLD) $(LDFLAGS) -o $(GS_XE)
+	$(ECHOGS_XE) -w $(ldt_tr) -n - $(CCLD) $(LDFLAGS) $(LDFLAGS_SO) -o $(GS_XE)
 	$(ECHOGS_XE) -a $(ldt_tr) -n -s $(PSOBJ)gsromfs$(COMPILE_INITS).$(OBJ) $(PSOBJ)gs.$(OBJ) -s
 	cat $(ld_tr) >>$(ldt_tr)
 	$(ECHOGS_XE) -a $(ldt_tr) -s - $(EXTRALIBS) $(STDLIBS)
